2024-05-31 05:14:29,402 INFO    MainThread:30976 [wandb_setup.py:_flush():76] Current SDK version is 0.17.0
2024-05-31 05:14:29,402 INFO    MainThread:30976 [wandb_setup.py:_flush():76] Configure stats pid to 30976
2024-05-31 05:14:29,402 INFO    MainThread:30976 [wandb_setup.py:_flush():76] Loading settings from /home/alex/.config/wandb/settings
2024-05-31 05:14:29,402 INFO    MainThread:30976 [wandb_setup.py:_flush():76] Loading settings from /home/alex/ECE239AS/DQN/wandb/settings
2024-05-31 05:14:29,402 INFO    MainThread:30976 [wandb_setup.py:_flush():76] Loading settings from environment variables: {}
2024-05-31 05:14:29,402 INFO    MainThread:30976 [wandb_setup.py:_flush():76] Applying setup settings: {'_disable_service': False}
2024-05-31 05:14:29,402 INFO    MainThread:30976 [wandb_setup.py:_flush():76] Inferring run settings from compute environment: {'program': '<python with no main file>'}
2024-05-31 05:14:29,402 INFO    MainThread:30976 [wandb_setup.py:_flush():76] Applying login settings: {}
2024-05-31 05:14:29,402 INFO    MainThread:30976 [wandb_init.py:_log_setup():520] Logging user logs to /home/alex/ECE239AS/DQN/wandb/run-20240531_051429-nr9ocqse/logs/debug.log
2024-05-31 05:14:29,402 INFO    MainThread:30976 [wandb_init.py:_log_setup():521] Logging internal logs to /home/alex/ECE239AS/DQN/wandb/run-20240531_051429-nr9ocqse/logs/debug-internal.log
2024-05-31 05:14:29,402 INFO    MainThread:30976 [wandb_init.py:init():560] calling init triggers
2024-05-31 05:14:29,402 INFO    MainThread:30976 [wandb_init.py:init():567] wandb.init called with sweep_config: {}
config: {}
2024-05-31 05:14:29,402 INFO    MainThread:30976 [wandb_init.py:init():585] re-initializing run, found existing run on stack: b8fbms6q
2024-05-31 05:14:29,403 INFO    MainThread:30976 [wandb_run.py:_finish():2103] finishing run ucla-alex/racing-car-dqn/b8fbms6q
2024-05-31 05:14:29,404 INFO    MainThread:30976 [jupyter.py:save_history():445] not saving jupyter history
2024-05-31 05:14:29,404 INFO    MainThread:30976 [jupyter.py:save_ipynb():373] not saving jupyter notebook
2024-05-31 05:14:29,404 INFO    MainThread:30976 [wandb_init.py:_jupyter_teardown():448] cleaning up jupyter logic
2024-05-31 05:14:29,404 INFO    MainThread:30976 [wandb_run.py:_atexit_cleanup():2343] got exitcode: 0
2024-05-31 05:14:29,404 INFO    MainThread:30976 [wandb_run.py:_restore():2326] restore
2024-05-31 05:14:29,404 INFO    MainThread:30976 [wandb_run.py:_restore():2332] restore done
2024-05-31 05:14:33,965 INFO    MainThread:30976 [wandb_run.py:_footer_history_summary_info():3994] rendering history
2024-05-31 05:14:33,966 INFO    MainThread:30976 [wandb_run.py:_footer_history_summary_info():4026] rendering summary
2024-05-31 05:14:33,973 INFO    MainThread:30976 [wandb_run.py:_footer_sync_info():3953] logging synced files
2024-05-31 05:14:34,237 INFO    MainThread:30976 [wandb_init.py:init():610] starting backend
2024-05-31 05:14:34,237 INFO    MainThread:30976 [wandb_init.py:init():614] setting up manager
2024-05-31 05:14:34,238 INFO    MainThread:30976 [backend.py:_multiprocessing_setup():105] multiprocessing start_methods=fork,spawn,forkserver, using: spawn
2024-05-31 05:14:34,238 INFO    MainThread:30976 [wandb_init.py:init():622] backend started and connected
2024-05-31 05:14:34,244 INFO    MainThread:30976 [wandb_run.py:_label_probe_notebook():1328] probe notebook
2024-05-31 05:14:34,244 INFO    MainThread:30976 [wandb_run.py:_label_probe_notebook():1338] Unable to probe notebook: 'NoneType' object has no attribute 'get'
2024-05-31 05:14:34,244 INFO    MainThread:30976 [wandb_init.py:init():711] updated telemetry
2024-05-31 05:14:34,809 INFO    MainThread:30976 [wandb_init.py:init():744] communicating run to backend with 90.0 second timeout
2024-05-31 05:14:35,372 INFO    MainThread:30976 [wandb_run.py:_on_init():2396] communicating current version
2024-05-31 05:14:35,483 INFO    MainThread:30976 [wandb_run.py:_on_init():2405] got version response 
2024-05-31 05:14:35,483 INFO    MainThread:30976 [wandb_init.py:init():795] starting run threads in backend
2024-05-31 05:14:36,844 INFO    MainThread:30976 [wandb_run.py:_console_start():2374] atexit reg
2024-05-31 05:14:36,844 INFO    MainThread:30976 [wandb_run.py:_redirect():2229] redirect: wrap_raw
2024-05-31 05:14:36,844 INFO    MainThread:30976 [wandb_run.py:_redirect():2294] Wrapping output streams.
2024-05-31 05:14:36,844 INFO    MainThread:30976 [wandb_run.py:_redirect():2319] Redirects installed.
2024-05-31 05:14:36,844 INFO    MainThread:30976 [wandb_init.py:init():838] run started, returning control to user process
2024-05-31 05:14:36,844 INFO    MainThread:30976 [wandb_run.py:_config_callback():1376] config_cb None None {'lr': 0.00025, 'gamma': 0.95, 'buffer_size': 100000, 'batch_size': 32, 'loss_fn': 'mse_loss', 'device': 'cuda', 'seed': 42, 'save_path': './runs/DoubleDQN_SoftUpdates/run0'}
